plugins/dynamic-home-page/catalog-info.yaml

```diff
@@ -0,0 +1,61 @@
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: janus-idp-dynamic-home-page
+  title: Dynamic Home Page plugin
+  description: Home Page plugin based on the upstream home [page] plugin that can be extended via (dynamic) plugins.
+  annotations:
+    backstage.io/source-location: url:https://github.com/janus-idp/backstage-showcase/tree/main/plugins/dynamic-home-page
+    #backstage.io/source-location: file:../../plugins/dynamic-home-page/REQUIRED_TO_BUILD_THE_TECHDOCS_LOCALLY
+    backstage.io/view-url: https://github.com/janus-idp/backstage-showcase/blob/main/plugins/dynamic-home-page/catalog-info.yaml
+    backstage.io/edit-url: https://github.com/janus-idp/backstage-showcase/edit/main/plugins/dynamic-home-page/catalog-info.yaml
+    backstage.io/techdocs-ref: dir:.
+    github.com/project-slug: janus-idp/backstage-showcase
+    github.com/team-slug: janus-idp/rhtap
```

Comment:
why rhtap? should this be rhdh?

Reply:
Good catch, I changed it to `janus-idp/maintainers-plugins`, what we used in backstage-plugins as well.